src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -1031,8 +1031,7 @@ export default function CDPipeline({
             })
             .catch((error: ServerErrors) => {
                 // 412 is for linked pipeline and 403 is for RBAC
-                // 422 is for deployment window
-                if (!force && error.code != 403 && error.code != 412 && error.code != 422) {
+                if (!force && error.code != 403 && error.code != 412) {
```

Comment:
Can we add comment: For now we are removing check for error code 422 which is of deployment window, so in that case force delete modal would be shown and this should be done at BE and when done we will revert our changes